tests/legacy-cli/e2e/tests/ivy/ivy-lazy-load.ts

```diff
@@ -0,0 +1,88 @@
+/**
```

Comment:
The tests are removed in the previous commit, but only re-added here. They should be moved in this commit instead.